.github/dockerfiles/ov_build/ubuntu_22_04_x64_nvidia/Dockerfile

```diff
@@ -12,11 +13,11 @@ ENV DEBIAN_FRONTEND="noninteractive" \
     TZ="Europe/London"
 
 RUN apt-get update && \
-    apt-get install software-properties-common && \
+    apt-get install --fix-missing software-properties-common && \
```

Comment:
why do we need `--fix-missing`? I saw such recommendation to resolve some previous installation mistakes, which we actually don't expect in our docker files.

Reply:
Just doing a debug, commited by mistake, will remove.